app/main/controllers/activity/RTMediaActivity.php

```diff
@@ -122,7 +122,7 @@ function media( $media, $type = 'activity' ) {
 				$thumbnail_id = $media->media_id;
 				if ( $thumbnail_id ) {
 					list( $src, $width, $height ) = wp_get_attachment_image_src( $thumbnail_id, apply_filters( 'rtmedia_activity_image_size', 'rt_media_activity_image' ) );
-					$html = '<img alt="' . esc_attr( $media->media_title ) . '" src="' . esc_url( $src ) . '" />';
+					$html = '<img alt="' . esc_attr( $media->media_title ) . '" src="' . set_url_scheme( $src ) . '" />';
```

Comment:
@pradeep910 

That code is added by checking the code from "wp-admin/includes/media.php:2740".
Because in media library, it was not breaking.

We should use `esc_url`. But WP is inserting name with special character.So it is the issue from WP.

The issue is reported here: https://core.trac.wordpress.org/ticket/15955 and https://core.trac.wordpress.org/ticket/41353